backend/vertical-healthcare_ch/forum_datenaustausch_ch.invoice_450_request/src/test/resources/de/metas/vertical/healthcare_ch/forum_datenaustausch_ch/invoice_450/Invoice450RequestConversionServiceTest.snap

```diff
@@ -1,3 +1,8 @@
+de.metas.vertical.healthcare_ch.forum_datenaustausch_ch.invoice_450.Invoice450RequestConversionServiceTest.exampleFile_Abrechnung_Praktischer_Arzt_TARMED_450_with_augment=[
```

Comment:
can we save this XML using some human friendly formatting.

why?
Easy to follow for the human in case there is some change and the test fails.
Having 1 huge line would be super hard for developer to understand where is the actually change.

Reply:
makes total sense, however all the existing tests in `invoice_440` use this no-formatting approach. I can look into how to save the snapshot into a human readable format.

Reply:
not sure, loud thinking,
i think here de.metas.vertical.healthcare_ch.forum_datenaustausch_ch.invoice_xversion.JaxbUtil#marshal 
u shall add a "prettyFormat" parameter 
and somehow configure the marshaller

Reply:
Changed to do testing against pretty-print formatted XMLs. Production should still use non-formatted XMLs.